crates/lair_keystore/Cargo.toml

```diff
@@ -14,6 +14,7 @@ edition = "2018"
 futures = "0.3"
 ghost_actor = "0.3.0-alpha.1"
 lair_keystore_api = { version = "=0.0.1-alpha.12", path = "../lair_keystore_api" }
+serde_yaml = "0.8"
```

Comment:
Once these keyfiles are properly encrypted, they will just be blobs of raw bytes, and maybe yaml doesn't make as much sense...

My suggestion might be to do away with any encoding, and just interpret the raw bytes of a file like:

```
[0-32) = salt bytes (these first 32 bytes should be all zeros while we're not encrypting anything)
[32-64) = 32 byte ed25519 public key
[64-96) = 32 byte seed used to generate ed25519 private key
```